### DISCUSSION TITLE
Respawn timer on Slayer tasks

### DISCUSSION THREAD

**Discussion by Z7-z** [runelite/runelite#16773] [Ideas]
Currently only respawn timers are available if you tag all kinds of different NPCs and add them to NPC Indicators, however with the wonderful Slayer plugin it highlights all the mobs on the task you’re doing, making NPC indicators unnecessary, but then won’t have the respawn timer. You also wouldn’t want to add every task to NPC indicators since you’ll have so many mobs permanently highlighted that you don’t want to see unless you’re on a task, if slayer tasks could have respawn timers added that would be amazing

**Comment by Nightfirecat** [comment]
Duplicate of https://github.com/runelite/runelite/issues/3730
